src/modules/search/search.controller.js

```
@@ -1,0 +1,86 @@
+
+    if (!text) {
+      return res.status(400).json({ 
+        success: false, 
+        message: 'Tham số văn bản là bắt buộc' 
+      });
+    }
+
+    const suggestions = await searchService.getProductSuggestions(
+      text,
+      parseInt(limit) || 5
+    );
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)